### ISSUE TITLE
Examples in the readme

### ISSUE THREAD

**Issue by danyaljj** [ruippeixotog/scala-scraper#12]
Are you sure your examples in the readme accord to your latest version? I have problems with them. Like this one: 

```
val items: Seq[Element] = doc >> elements(".item")
```

**Comment by ruippeixotog** [comment]
Hi @danyaljj! It is indeed a mistake in the examples. I don't think that line ever worked that way unless you imported `JavaConversions`, as `doc >> elements(".item")` has type `Elements`, which extends `ArrayList<Element>`.

I have just fixed the example in a822622.

**Comment by danyaljj** [comment]
I see thanks. 
BTW, if you give examples based on html snippets (rather than loading web pages), your examples would become much more illuminating for users who have no familiarity with your code's operators, etc.

**Comment by ruippeixotog** [comment]
Hmm, ok, nice point. Do you think the comments accompanying the examples are not enough for users to understand what does the example do? If that's so, maybe I'll follow your suggestion and make the examples extract content from an inline HTML snippet.

**Comment by danyaljj** [comment]
I don't have any specific examples. I noticed it when I was trying to understand your examples, and I found myself so much in need to seeing inline html snippets on which your operations are being applied.

**Comment by akamaus** [comment]
@danyaljj, I've recently found ScalaScraper and I found test specs to be quite useful source of examples. As a bonus it's easy for library maintainer to support them in working state.

**Comment by ruippeixotog** [comment]
Hi all! In the latest version 2.0.0-RC1 I improved considerably the examples in the README. I'm now using [tut](https://github.com/tpolecat/tut), which actually compiles the examples in the Markdown files and writes the output of each command as if it was written in a REPL. I'm generating and checking the output of tut on Travis, so the examples are always kept up-to-date.

Additionally, all examples run against a [sample HTML document](https://github.com/ruippeixotog/scala-scraper/blob/master/core/src/test/resources/example.html) provided in the repo, with which users can follow and understand better the results.

I took a long time, but I think I have finally addressed the difficulties you mentioned. I'll close this issue now, but feel free to open it again if you think more can be done. Thank you!
